### PR TITLE
Netfix Branch Pull Request

### PR DESCRIPTION
**Pull request by kamal2222ahmed**
This pull request is for:

1. Meaningful network name
2. reverting the DB_NAME to db
3. Fixing the Connection errors

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,7 +25,10 @@ services:
       timeout: 5s
       retries: 10
     environment:
+      - DB_HOST=db
```
Comment (ludeknovy) [inline docker-compose.yml:28]: I dont think this is needed

**Comment by ludeknovy** [comment]
And does it solve the https://github.com/ludeknovy/jtl-reporter/issues/46 for you?

**Comment by kamal2222ahmed** [comment]
yes, it does solve the #46

File: docker-compose.yml (hunk 1 above)
Comment (ludeknovy) [reply docker-compose.yml:28]: @kamal2222ahmed please remove this and I think we are good to merge it.

Comment (ludeknovy) [reply docker-compose.yml:28]: @kamal2222ahmed  could you please resolve this?

**Comment by ludeknovy** [comment]
Feel free to re-open once the comments are resolved

**Pull request closed without merging**